workshops/custom_link_shortener/README.md

```diff
@@ -0,0 +1,227 @@
+---
+name: 'Custom Link Shortener'
+description: 'Build your own custom link shortener to save time remembering links.'
+author: '@shreyfirst'
+img: 'https://cloud-e15trekvp.vercel.app/0image.png'
+
+---
+
+In this workshop, you will learn how to use:
+
+- [Express JS servers](https://expressjs.com/)
+- [Key-value databases](https://en.wikipedia.org/wiki/Key%E2%80%93value_database)
+- [HTML forms](https://www.w3schools.com/html/html_forms.asp)
+
+You will be making an [API](https://en.wikipedia.org/wiki/API) that allows someone to submit a form with a link and in return, get a personalized custom short link to use in replacement of that initial long link.
+
+**Difficulty/interest level:** This workshop is designed for all level hackers in [HTML](https://www.w3schools.com/html/) and [Javascript](https://www.w3schools.com/js/). You'd be interested in this if you want to learn how to use Express JS or build a link shortener. You do not need to buy a [domain](https://en.wikipedia.org/wiki/Domain_name) for this as you can use repl.it's custom project domain.
+
+**Average time for completion:** You can complete this in <u>15-20 minutes</u>. You will need the following tools:
+
+- [Repl.it](https://repl.it/) online coding platform (no account necessary, but highly recommended)
```

Comment:
Actually as of a few weeks ago, repl.it requires you to create an account. It's probably better if you remove the stuff in the parentheses altogether